src/components/inputs/answers/SimpleTextInput.jsx

```diff
@@ -76,7 +76,7 @@ function SimpleTextInput(props) {
             )}
 
             <input
-                type="text"
+                type={item.type === 'number' ? 'number' : 'text'}
```

Comment:
```suggestion
                type={item.type === 'NUMBERBOX' ? 'number' : 'text'}
```
- Tipo diferente do definido na API;